### DISCUSSION TITLE
Lowhanging fruit:  some basic colon commands

### DISCUSSION THREAD

**Discussion by PopeNobody** [JetBrains/ideavim#538] [Ideas]
I've been wanting to hack in ideavim, and I've been thinking about implementing some of the low-hanging-fruit in the colon mode commands.

I've been using vim for a long time, and I have some pretty strong habits ... like typing :make and wanting the thing I'm working on to do something ... I tend to do things like 

:set makeprg=vi_perl\ % 

when I'm working on a perl script ( where vi_perl runs forks, runs perl, and hacks the perl stack traces into 

perl.pl:5: Oh, shit, I'm gonna die()

so vim can recognize them like gcc errors.

So I'd like to make :make in ideaj run gradle, like F9.

Also, I'd love to be able to say :tag Identifier and jump to it.

So I wanted to see if anybody thought this was a terrible idea before I started .. and if somebody could give me a hint on a firename in which to look for the colon mode commands, it might save me some time.

**Comment by AlexPl292** [comment]
Hi there! Supporting `make` option seems to be a good idea.
You can take a look at `OptionService` to see how the options are implemented.
